src/simulation/LoadGenerator.cpp

```diff
@@ -168,34 +170,84 @@ void
 LoadGenerator::reset()
 {
     mAccounts.clear();
+    mAccountsInUse.clear();
+    mAccountsAvailable.clear();
+    mCreationSourceAccounts.clear();
+    mLoadTimer.reset();
     mRoot.reset();
     mStartTime.reset();
     mTotalSubmitted = 0;
     mWaitTillCompleteForLedgers = 0;
     mFailed = false;
+    mStarted = false;
+    mInitiaAccountsCreated = false;
 }
 
 // Schedule a callback to generateLoad() STEP_MSECS milliseconds from now.
 void
 LoadGenerator::scheduleLoadGeneration(GeneratedLoadConfig cfg)
 {
+    std::optional<std::string> errorMsg;
     // If previously scheduled step of load did not succeed, fail this loadgen
     // run.
     if (mFailed)
     {
-        CLOG_ERROR(LoadGen, "Load generation failed, ensure correct "
-                            "number parameters are set and accounts are "
-                            "created, or retry with smaller tx rate.");
+        errorMsg = "Load generation failed, ensure correct "
+                   "number parameters are set and accounts are "
+                   "created, or retry with smaller tx rate.";
+    }
+    // During account creation, we can create up to
```

Comment:
Can we add a comment above explaining nAccount, txRate, and batchSize relationship briefly?